htdocs/variants/tpl/productattributevalueline_view.tpl.php

```diff
@@ -38,6 +38,9 @@
 	exit;
 }
 
+'@phan-var-force CommonObject $this
```

Comment:
Having this into code looks strange.
Should be into comment ?

Reply:
These are implemented as string literals due to the limitation of php-ast being unable to parse inline doc comments.
From the docs:
https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#inline-type-checks-via-string-literals

Mostly needed in view files and some rare cases in code.

Reply:
However on file supplier_proposal.class.php, we have
// @phan-suppress-next-line PhanPluginSuspiciousParamOrde
So it seems possible to have phan tags in doc.